effectivity/README.md

```diff
@@ -0,0 +1,146 @@
+--- 
+layout: pattern
+title: Effectivity
+folder: effectivity
+permalink: /patterns/effectivity/
+categories: Creational
+language: en 
+tags:
+ - Data access
+---
+
+## Name / classification
+
+Effectivity.
+
+## Also known as
+
+Effective Dating, Temporal Database Design.
+
+## Intent
+
+Add a time period to an object to show when it is effective.
+
+## Explanation
+
+Real-world example
+
+> An individual is employed at different companies over different periods of time during their 
+> life. It is useful to know if their employment at a certain position is effective during 
+> different points in time.
+
+In plain words
+
+> Effectivity pattern lets you specify a time period over which an object is effective.
+
+[Martin Fowler](https://martinfowler.com/eaaDev/Effectivity.html) says
```

Comment:
I realise there is not a Wikipedia page for this design pattern. But the maintainer would probably prefer if this was captioned Wikipedia says still.

Reply:
I've decided to have it like this because it isnt wikipedia, and is instead an exerpt that Martin Fowler has written.